README.md

```diff
@@ -112,6 +112,22 @@ const base64PreciseProofs = new PreciseProofs('base64');
 const base64Valid = base64PreciseProofs.isValidField(base64Proof, base64RootHash)
 // base64Valid  === true 
 ```
+## About Transformation Between Compacts and Readable Name of Proof
+TransformHelper can be used to achieve this target
+First of all, you shoud install protobufjs to generate json definition from proto file.
+ 
+```
+pbjs example.proto -t json -p "/usr/local/include" -o examples.json
+```
+To transform between Compacts format and Readable Name, you should provide generated json file and you should know the package name and the root message type defined in proto file.
+
+```
+import {TransformHelper} from "readable_compacts_transform_helper";
```

Comment:
Adding a few things on naming in the comments below to keep terminology in line with the go code base: 
Can you rename it to: PropertyTransformHelper?
The package `readable_compacts_transform_helper` seems a bit long. Why not make it part of the `precise-proofs` package?